frontend/src/app/main/data/messages.cljs

```diff
@@ -93,18 +95,17 @@
           (rx/of hide))))))
 
 (defn error
-  ([content] (error content {}))
-  ([content {:keys [timeout] :or {timeout default-timeout}}]
-   (show {:content content
-          :type :error
-          :position :fixed
-          :timeout timeout})))
+  ([content] ((show {:content content
```

Comment:
`((show...` looks wrong
`([content...` should be `[content]` on a function with single arity